python/grass/temporal/c_libraries_interface.py

```diff
@@ -743,10 +743,8 @@ def _read_raster_info(name, mapset):
         range = libraster.FPRange()
         libraster.Rast_init_fp_range(byref(range))
         ret = libraster.Rast_read_fp_range(name, mapset, byref(range))
-        if ret < 0:
-            logging.error(_("Unable to read range file"))
-            return None
-        if ret == 2:
+        if ret < 0 or ret == 2:
+            logging.warning(_("Unable to read range file"))
```

Comment:
Setting the values seems fine, but the warning text is technically not correct for ret == 2:
https://grass.osgeo.org/programming7/defs_2raster_8h.html#a9efe2d261c71822b99f546c6d740395c
I know this is fairly minor, but still...

Reply:
Right. Addressed in [faab6c1](https://github.com/OSGeo/grass/pull/2382/commits/faab6c174692c2e5860d01099fbe4a20fe9b4af4). Could probably be solved with less code duplication, but I am not sure it is worth doing bigger changes here...